File: dataTrainingServer/Models/ClassModel.py
class ClassModel:

    def __init__(self, label, id = ''):
        self._label = label
        self._id = id

    def get_label(self):
        return self._label
    
    def get_id(self):
        return self._id
    
    def set_label(self, label):
        self._label = label

    def set_id(self, id):
        self._id = id

File: applicationServer/demo/src/main/java/com/example/demo/models/ClassModel.java
package com.example.demo.models;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table
public class ClassModel {
    @Id
    @Column(name = "id")
    private String id;
    @Column(name = "label")
    private String label;

    public ClassModel(String id, String label) {
        this.id = id;
        this.label = label;
    }

    public ClassModel(String label) {
        this.label = label;
    }

    public ClassModel() {
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getLabel() {
        return label;
    }

    public void setLabel(String label) {
        this.label = label;
    }
}
